Guidelines.md

```diff
@@ -211,11 +216,9 @@ https://api.contoso.com/v1.0/items?url=https://resources.contoso.com/shoes/fancy
 ```
 
 ### 7.2 URL length
-The HTTP 1.1 message format, defined in RFC 7230, in section [3.1.1][rfc-7230-3-1-1], defines no length limit on the Request Line, which includes the target URL. From the RFC:
+The HTTP 1.1 protocol, defined in RFC 2616, in section [3.2.1][rfc-2616-3-2-1], defines no URL length limit. From the RFC:
 
-> HTTP does not place a predefined limit on the length of a
-   request-line. [...] A server that receives a request-target longer than any URI it wishes to parse MUST respond
-   with a 414 (URI Too Long) status code.
+> The HTTP protocol does not place any a priori limit on the length of a URL. Servers MUST be able to handle the URL of any resource they serve, and SHOULD be able to handle URLs of unbounded length if they provide GET-based forms that could generate such URLs. A server SHOULD return 414 (Request-URL Too Long) status if a URL is longer than the server can handle.
```

Comment:
:exclamation: I believe this was correct before (it reverts a change made in #41)